assets/js/components/Work/UIWorkCardItem.jsx

```diff
@@ -1,64 +1,78 @@
-import React from "react";
+import React, { useContext } from "react";
 import PropTypes from "prop-types";
 import { Link } from "react-router-dom";
 import { setVisibilityClass, formatDate } from "../../services/helpers";
+import { IIIFProvider, IIIFContext } from "../../components/IIIF/IIIFProvider";
 
-const WorkCardItem = ({ work }) => {
+const WorkCardItem = ({
+  id,
+  representativeImage,
+  title,
+  descriptiveMetadata,
+  workType,
+  visibility,
+  published,
+  accessionNumber,
+  fileSets,
+  manifestUrl,
+  updatedAt,
+}) => {
+  const iiifServerUrl = useContext(IIIFContext);
   return (
     <div className="card " data-testid="ui-workcard">
       <div className="card-image">
         <figure className="image is-4by3">
-          <Link to={`/work/${work.id}`}>
+          <Link to={`/work/${id}`}>
             <img
               src={`${
-                work.representativeImage
-                  ? work.representativeImage + "/full/1280,960/0/default.jpg"
-                  : "/images/1280x960.png"
+                representativeImage.id
+                  ? iiifServerUrl +
+                    representativeImage.id +
+                    "/square/500,500/0/default.jpg"
+                  : representativeImage + "/full/1280,960/0/default.jpg"
               }`}
               data-testid="image-work"
-              alt={work.title}
+              alt={title}
             />
           </Link>
         </figure>
       </div>
       <div className="card-content">
         <h3 className="title is-size-4">
-          {work.descriptiveMetadata.title
-            ? work.descriptiveMetadata.title
-            : "Untitled"}
+          {descriptiveMetadata.title ? descriptiveMetadata.title : "Untitled"}
```

Comment:
Along the lines of above, let's figure out what `title` is the official title which should be used.  Maybe ping @kdid ?